Library/Formula/silk.rb

```diff
@@ -17,8 +15,14 @@ def install
                           "--enable-ipv6",
                           "--enable-data-rootdir=#{var}/silk"
     system "make"
-    system "make install"
+    system "make", "install"
 
     (var+"silk").mkpath
   end
+
+  test do
+    input = test_fixtures("test.pcap")
+    output = `yaf --in #{input} | #{bin}/rwipfix2silk | #{bin}/rwcount --no-titles --no-column`
```

Comment:
`shell_output()` okay, as for switching to temp files, I'd rather give up at that point, sorry.